engine/src/main/scala/org/scalarules/dsl/core/glossaries/Glossary.scala

```diff
@@ -11,7 +11,7 @@ import scala.language.experimental.macros
   * Utility base class for collecting and namespacing `Fact`s. You can extend this class, define facts in it and receive a
   * utility collection of all facts declared in your class.
   */
-class MacroGlossary {
+class Glossary {
   def defineFact[A](): SingularFact[A] = macro FactMacros.defineFactMacroImpl[A]
   def defineFact[A](description: String = "No description"): SingularFact[A] = macro FactMacros.defineFactMacroWithDescriptionImpl[A]
```

Comment:
I understand the need for two methods. I was arguing that the default "no description" for the method which has a description is perhaps not neccessary. Not supplying a description will not fill in "no description", but rather call the no-arg method. And leaving it like this might cause confusion.
Unless of course I am missing some vital macro knowledge here.